### PR TITLE
NT-192 Speech To Text

### PR DESCRIPTION
**Pull request by PatrickSlag**
Tested Browser Support:
- [x] Edge
- [x] Chrome
- [ ] ~~Firefox~~ - Not supported
- [x] Safari

### CODE REVIEW

**Comment by PatrickSlag** [comment]
If somebody could test this in safari. it should work but just to be sure

**Comment by jortdr** [comment]
Tested on Safari Mac and on Safari iPad. Works, but not for the second try.

File: Assets/Scripts/Presenters/SpeechPresenter.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            if (debugMode)
+            {
+                debugText.text = result;
+                Debug.Log("final: " + result);
+            }
+        }
+
+        public void StartRecognition()
+        {
+            StartSpeechRecognition();
```
Comment (jortdr) [inline Assets/Scripts/Presenters/SpeechPresenter.cs]: Should check if platform is supported and return if not supported.

File: Assets/Plugins/TextToSpeech.jslib
Comment (jortdr) [inline Assets/Plugins/TextToSpeech.jslib:1]: format this file?

**Review by jortdr — CHANGES_REQUESTED** [review]

**Review by VarioCode — DISMISSED** [review]

**Review by jortdr — APPROVED** [review]

**Review by VarioCode — APPROVED** [review]

**Pull request merged**